src/commands/importExportEntities/entities/assets.ts

```diff
@@ -1,25 +1,93 @@
-import { AssetContracts } from "@kontent-ai/management-sdk";
+import { AssetContracts, ManagementClient } from "@kontent-ai/management-sdk";
 import JSZip from "jszip";
 
 import { serially } from "../../../utils/requests.js";
-import { EntityDefinition } from "../entityDefinition.js";
+import { EntityDefinition, ImportContext } from "../entityDefinition.js";
 
-export const assetsExportEntity: EntityDefinition<ReadonlyArray<AssetContracts.IAssetModelContract>> = {
+const assetsBinariesFolderName = "assets";
+const createFilePrefix = (asset: AssetContracts.IAssetModelContract) => `${asset.id}-`;
```

Comment:
Because I thought that I would need the length of the prefix to strip it when importing, but it turns out I can get the name from the asset entity itself so this is not needed and therefore, I can make it `createFileName` instead. 😄